quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/PersistedConfigSource.java

```diff
@@ -17,37 +17,38 @@
 
 package org.keycloak.quarkus.runtime.configuration;
 
-import java.io.File;
 import java.io.IOException;
-import java.nio.file.Path;
+import java.io.InputStream;
+import java.net.URL;
 import java.util.Collections;
+import java.util.HashMap;
 import java.util.Map;
+import java.util.Properties;
 
 import io.smallrye.config.PropertiesConfigSource;
-import io.smallrye.config.common.utils.ConfigSourceUtil;
 import org.keycloak.quarkus.runtime.Environment;
 
 /**
  * A {@link org.eclipse.microprofile.config.spi.ConfigSource} based on the configuration properties persisted into the server
  * image.
  */
-public class PersistedConfigSource extends PropertiesConfigSource {
+public final class PersistedConfigSource extends PropertiesConfigSource {
 
     public static final String NAME = "PersistedConfigSource";
-    static final String KEYCLOAK_PROPERTIES = "persisted.properties";
+    public static final String KEYCLOAK_PROPERTIES = "/META-INF/keycloak-persisted.properties";
```

Comment:
can we also call this KC_PERSISTED_PROPERTIES or sth so we don't get confused? bc. we also have keycloak.properties file :)

Reply:
Done